libs/libm/Kconfig

```diff
@@ -3,11 +3,12 @@
 # see the file kconfig-language.txt in the NuttX tools repository.
 #
 
-#menu "Math Library Support"
+choice
+	prompt "Select math library"
+	default LIBM_NONE
```

Comment:
Only one question to `default` value. Before selection was defaulted to what is called `LIBM_TOOLCHAIN` here, so maybe
```
default LIBM_NONE if DEFAULT_SMALL
default LIBM_TOOLCHAIN if !DEFAULT_SMALL
```
?

Reply:
good idea

Reply:
Done.